extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -103,10 +103,22 @@ export class DatabaseSettingsPage extends BasePage {
 				value: constants.IpAddressInfoLabel
 			}).component();
 
+		// regex for validation
+		let ipRegex = /(^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$)/;
+
 		//Start IP Address Section:
 
 		this._startIpAddressTextbox = view.modelBuilder.inputBox().withProperties(<azdata.InputBoxProperties>{
-			inputType: 'text'
+			inputType: 'text',
+			required: true,
+			validationErrorMessage: localize('deployAzureSQLDB.DBIpInvalidError', "Invalid IP address format")
```

Comment:
Put these in the localizedConstants file so you don't have to have multiple localize calls for the same string